src/cpp/src/image_generation/schedulers/euler_discrete.cpp

```diff
@@ -319,5 +319,13 @@ void EulerDiscreteScheduler::add_noise(ov::Tensor init_latent, std::shared_ptr<G
     }
 }
 
+void EulerDiscreteScheduler::set_timesteps_with_sigma(std::vector<float> sigma, float mu) {
+    OPENVINO_THROW("EulerDiscreteScheduler doesn't support set_timesteps_with_sigma");
+}
+
+float EulerDiscreteScheduler::calculate_shift(size_t image_seq_len) {
+    OPENVINO_THROW("EulerDiscreteScheduler doesn't support `calculate_shift` method");
```

Comment:
maybe we can keep such methods which are supported only by selected schedulers, to be implemented in base class with default implementations throwing an exception.